### PR TITLE
Idea.

### PR DESCRIPTION
**Pull request by pranavi79**
On theme eradicating hunger, we will be developing an app where NGO's /individuals can directly contact volunteers for distribution of food.
Our team:
1.pranavi.sharma79@gmail.com
2.sahashamoyeeta@yahoo.com
3.bhavikhator@gmail.com

# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

[*] Registration Pull request.

[] Submission Pull request.

## Checklist ✅  

- [] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [] I have used the Markdown file template to add my information for the Hackathon.
- [] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [] I understand I should not tag or add a reviewer to this Pull Request.
- [] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/teams/GrowTogetherteam [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+_**University-Name**_ : 
+
+_**country**_ :
+ 
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :  
+
```
Comment (Uyadav207) [inline content/teams/GrowTogetherteam]: @pranavi79 , The Template is empty. I have to close this Pull Request. please follow the instruction carefully.

Comment (pranavi79) [reply content/teams/GrowTogetherteam]: I'm so sorry sir, I was in process of making those changes(I was following the steps step by step), I have made required changes.
Yours Respectfully,
Pranavi
Grow-Together team

File: content/teams/GrowTogether/index.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ---
 type: "team"                   
-title: DemoTeam
-name: "DemoTeam"
-github-link: "https://github.com/ "
-description: "hello I am Bot Developer"
-Team-Quote: ""
-Team_Tech_Stack: ""
+title: GrowTogether app
+name: "GrowTogether"
```
Comment (Uyadav207) [inline content/teams/GrowTogether/index.md:4]: Title and name should be same as name of your team

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

File: content/teams/GrowTogether/index.md (hunk 2 above)
Comment (Uyadav207) [reply content/teams/GrowTogether/index.md:4]: Also in Idea Description : Please Describe your Idea for the project max 4-5 line.

Comment (Uyadav207) [reply content/teams/GrowTogether/index.md:4]: Also Give complete github link in github ID section for individual teammate

**Comment by Uyadav207** [comment]
@pranavi79 , You can make change here only no need to close the PR everyTime.

**Pull request closed without merging**